docs/osx/install.md

```diff
@@ -9,39 +9,63 @@ Most OS X developers should start by downloading and installing Xcode
 from the App Store.  It is a free IDE from Apple that provides
 all of the tools you need for working with wxWidgets.
 
-After Xcode is installed, download wxWidgets-{version}.tar.bz2 and then
+After Xcode is installed, download `wxWidgets-{version}.tar.bz2` and then
 double-click on it to unpack it to create a wxWidgets directory.
 
 Next use Terminal (under Applications, Utilities, Terminal) to access a command
-prompt.  Use cd to change directories to your wxWidgets directory and execute
+prompt.  Use `cd` to change directories to your wxWidgets directory and execute
 the following sets of commands from the wxWidgets directory.
 
     mkdir build-cocoa-debug
     cd build-cocoa-debug
     ../configure --enable-debug
     make
 
-Build the samples and demos
+It is recommended to use `-jN` option with the last command, where `N` is a
+number of the processors in your system (which can be checked using `sysctl -n
+hw.ncpu` command if you are not sure), as this will dramatically speed up the
+build on modern systems. So in practice you should use a command like this:
 
-    cd samples; make;cd ..
-    cd demos;   make;cd ..
+    make -j16
```

Comment:
Because `-j16` requires quite a high-end Mac system to be useful, and some people are likely to just copy-paste `-j16` without thinking how many cores or how much memory they have, I propose using a smaller figure in this example. Such as 6 or 8.

Reply:
OK, done, and added an extra warning about this, thanks.